### ISSUE TITLE
Version of HuggingFace

### ISSUE THREAD

**Issue by zhangry868** [epfml/collaborative-attention#2]
I am interested in reproducing reported results but met some issues. 

It seems that the package installation requires ver. 2.5.0, but run_glue.py is only compitable with later versions (e.g., ver 3.0.1).

Thanks!

**Comment by jbcdnr** [comment]
We ran everything using Transformers 2.x. It's quite unfortunate that we released our code just when they released the new version... I had made minor changes from the [run_glue.py](https://github.com/huggingface/transformers/blob/b42586ea560a20dcadb78472a6b4596f579e9043/examples/text-classification/run_glue.py) from 2.x but you are right, I pushed a version based on `run_glue.py` 3.x in the repo. 

I will push a fix soon. Thanks for reporting.
